scss/6-components/_footer.scss

```diff
@@ -75,7 +75,7 @@
   .cads-footer__company-info {
     @include cads-restrict-content-width();
 
-    color: $cads-footer__text-colour;
+    color: $cads-language__text__colour;
```

Comment:
This probably _does_ need to be specific to the footer. By using the shared `$cads-language__text__colour` I'm not sure how we'd handle themeing like with AdviserNet/Intranet as this colour needs to be reversed out in those cases.

Reply:
I see, good shout! I think if we keep them, we can perhaps put the header and footer vars into different files - that was the bit that made me remove them in first place